### PR TITLE
Update JetBrains Rider documentation

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**
Revised JetBrains Rider setup steps and clarified debug configurations. Replaced outdated .webp images with new .png files for newer and clearer screenshots.

### CODE REVIEW

**Review by skyace65 — CHANGES_REQUESTED** [review]
Please convert the pngs you created to webp. It's the standard format we're using with the docs going forward. The original images were definitely overcompressed but that's not an inherit flaw with webp, whoever made those just did that, you can make webps with lossless compression which is what we recommend. I would recommend using [squoosh ](https://squoosh.app/)to convert those images and just use the maximum lossless compression.

**Comment by khalidabuhakmeh** [comment]
I've converted the images to `webp` as specified. Apologies. I saw some other `png` files in the directory and thought they were either.

**Review by skyace65 — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

File: contributing/development/configuring_an_ide/rider.rst [hunk 1]
```diff
@@ -1,20 +1,25 @@
 .. _doc_configuring_an_ide_rider:
 
-Rider
+JetBrains Rider
 =====
```
Comment (mhilbrunner) [inline contributing/development/configuring_an_ide/rider.rst:4]:
```suggestion
JetBrains Rider
===============
```

**Comment by khalidabuhakmeh** [comment]
👋 Hi folks, I'm not sure what the error message means here, as I'm unfamiliar with sphinx. I could change the title of the page to "Rider" but that's technically not the name of the product.

File: contributing/development/configuring_an_ide/rider.rst [hunk 2]
```diff
@@ -47,23 +59,31 @@ if you want to debug the editor, you need to configure the debugger first.
...
 
 .. figure:: img/rider_attach_to_process.webp
    :align: center
+
+- You can find the Godot instance by searching for ``godot.editor`` and then clicking ``Attach with LLDB``
+
+.. figure:: img/rider_attach_to_process_dialog.webp
+   :align: center
+
+Please consult the `JetBrains Rider <https://www.jetbrains.com/rider/documentation/>`_ documentation for any specific information about the JetBrains IDE.
```
Comment (AThousandShips) [inline contributing/development/configuring_an_ide/rider.rst:89]: I think the issue is that you use the same name here, maybe:
```suggestion
Please consult the JetBrains Rider `documentation <https://www.jetbrains.com/rider/documentation/>`_ for any specific information about the JetBrains IDE.
```

Comment (khalidabuhakmeh) [reply contributing/development/configuring_an_ide/rider.rst]: Thank you @AThousandShips . I've fixed the issue and submitted another change.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Whew, that was a bit tricky. Merged. Thanks and congrats on your first merged contribution!

**Comment by khalidabuhakmeh** [comment]
Thank you everyone for your help—cheers and happy to contribute.

**Pull request merged**